scipy/optimize/_numdiff.py

```diff
@@ -334,7 +334,12 @@ def fun_wrapped(x):
     if np.any((x0 < lb) | (x0 > ub)):
         raise ValueError("`x0` violates bound constraints.")
 
-    h = _compute_absolute_step(rel_step, x0, method)
+    if abs_step is None:
+        h = _compute_absolute_step(rel_step, x0, method)
+    else:
+        # sign_x0 != 0 unlike np.sgn()
+        sign_x0 = (x0 >= 0).astype(float) * 2 - 1   
+        h = abs_step*sign_x0
```

Comment:
So this ignores `rel_step` if `abs_step` is not None, correct? Better check and raise an exception (silently swallowing incorrect input is not nice).